86.md

```diff
@@ -0,0 +1,88 @@
+NIP-86
+======
+
+Relay Management API
+--------------------
+
+`draft` `optional`
+
+Relays may provide an API for performing management tasks. This is made available as a JSON-RPC-like request-response protocol over HTTP, on the same URI as the relay's websocket.
+
+When a relay receives an HTTP(s) request with an `Authorization` header and a `Content-Type` header of `application/nostr+json+rpc` to a URI supporting WebSocket upgrades, it should parse the request as a JSON document with the following fields:
```

Comment:
```suggestion
When a relay receives an HTTP(s) request with a `Content-Type` header of `application/nostr+json+rpc` to a URI supporting WebSocket upgrades, it should parse the request as a JSON document with the following fields:
```

Content-Type should determine the code path, not Authorization. Without Authorization it should return a 401. The way it's written implies that a `Content-Type: application/nostr+json+rpc` with no Authorization should fall back to behaving like a regular relay.

Reply:
The request method also has to be POST.